drivers/display/Kconfig.sdl

```diff
@@ -80,4 +80,10 @@ config SDL_DISPLAY_TRANSPARENCY_GRID_CELL_COLOR_2
 	help
 	  The color of the even cells in the transparency grid.
 
+config SDL_DISPLAY_THREAD_PRIORITY
+	int "SDL display thread priority"
+	default 0
```

Comment:
I'd put the default priority way lower, I'd almost argue I'd put it to the lowest priority possible. Otherwise low priority threads calling draw functions would cause a priority inversion.

Is there a way to use priority inheritance to bump this priority dynamically when higher priority threads are waiting for this?

Reply:
@Finomnis note this is only for native_sim, where you are not constrained like in an embedded device. 
But more importantly note that this thread would only get unblocked if the thread that wants to draw has just been scheduled. So even though this thread has high priority, it is only running as a result of that requesting thread running.

Reply:
Ok :)

Reply:
Changed the default priority to `NUM_PREEMPT_PRIORITIES`.